cmake/common.cmake

```diff
@@ -446,7 +447,7 @@ function(add_e2e_test)
       NAME ${PARSED_ARGS_NAME}
       COMMAND
         ${PYTHON} ${PARSED_ARGS_PYTHON_SCRIPT} -b . --label ${PARSED_ARGS_NAME}
-        ${CCF_NETWORK_TEST_ARGS} -g ${PARSED_ARGS_GOV_SCRIPT} --constitution
+        ${CCF_NETWORK_TEST_ARGS} -g ${PARSED_ARGS_GOV_SCRIPT}
         ${PARSED_ARGS_CONSTITUTION} --consensus ${PARSED_ARGS_CONSENSUS}
```

Comment:
Should this also remove `${PARSED_ARG_CONSTITUTION}`? It appears to be left as a floating arg, no idea how it now gets processed but looks unintentional?

Reply:
No, this is on purpose, the idea being that it can now be used to pass in stuff like "--constitution fragment1.js --constitution fragment2.js"?

Reply:
Ah I see: like `CCF_NETWORK_TEST_DEFAULT_CONSTITUTION` does - `PARSED_ARG_CONSTITUTION` now contains multiple elements, including the `--constitution` bit. Makes sense!